vcGL/src/vcTextureHelper.cpp

```diff
@@ -246,6 +246,80 @@ udResult vcTexture_ResizePixels(const void *pPixels, uint32_t width, uint32_t he
   return result;
 }
 
+udResult vcTexture_GetPixelSize(vcTextureFormat format, size_t *pSize)
+{
+  static const int ByteCount[vcTextureFormat_Count] = {-1, 4, 8, 16, 2, 4};
+
+  if (format == vcTextureFormat_Count || pSize == nullptr)
```

Comment:
You should probably make this a bit more flexible to bad code. You can cast any number to `vcTextureFormat`, which means that you would want to ensure that only values between `vcTextureFormat_None` (inclusive) and `vcTextureFormat_Count` (exclusive) are allowed.